src/relayer/Relayer.ts

```diff
@@ -754,7 +756,10 @@ export class Relayer {
       );
       await this.evaluateFills(unfilledDeposits, lpFees, maxBlockNumbers, sendSlowRelays);
 
-      if (multiCallerClient.getQueuedTransactions(destinationChainId).length > 0) {
+      const pendingTxnAmount = this.config.tryMulticallChains.includes(destinationChainId)
+        ? tryMulticallClient.getQueuedTransactions(destinationChainId).length
+        : multiCallerClient.getQueuedTransactions(destinationChainId).length;
+      if (pendingTxnAmount > 0) {
```

Comment:
I was confused about `Amount` because it somehow made me think of "value txns" (i.e. txns w/ msg.value > 0). Thoughts on renaming it?

```suggestion
      const pendingTxns = this.config.tryMulticallChains.includes(destinationChainId)
        ? tryMulticallClient.getQueuedTransactions(destinationChainId).length
        : multiCallerClient.getQueuedTransactions(destinationChainId).length;
      if (pendingTxns > 0) {
```

Reply:
Done here: https://github.com/across-protocol/relayer/pull/1676/commits/e58634681f71aae14b79402c2bee084e6f7c7fb0